source/simulator/core.cc

```diff
@@ -130,7 +130,9 @@ namespace aspect
             this_mpi_process(mpi_communicator)
             == 0)),
 
-    computing_timer (pcout, TimerOutput::never,
+    computing_timer (mpi_communicator,
+                     pcout,
```

Comment:
Ok good to know :)
I was afraid this was an intentional decision to remove the barriers. I will update the PR according to the suggestions when there is some quiet time at the conference.